koku/api/settings/tags/mapping/utils.py

```diff
@@ -58,4 +65,48 @@ def resummarize_current_month_by_tag_keys(enabled_rows, schema_name):
             )
             delayed_summarize_current_month(schema_name, list(provider_uuids), provider_type)
 
-    return provider_type
+
+def retrieve_tag_rate_mapping(schema_name):
```

Comment:
I'm honestly not sure I understand what all of this sql is for. Why are we not using the ORM here?

Reply:
It creates a mapping of keys used in tag rates to cost model ids. I found it easier to build the query I needed, if that is a blocker I can spend more time figuring out the same thing with the ORM.